catroid/src/main/java/org/catrobat/catroid/stage/VisualPlacementActivity.java

```diff
@@ -173,10 +181,10 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
 		try {
 			String backgroundBitmapPath;
-			if (manualScreenshot.exists()) {
-				backgroundBitmapPath = manualScreenshot.getPath();
-			} else if (automaticScreenshot.exists()) {
+			if (automaticScreenshot.exists()) {
 				backgroundBitmapPath = automaticScreenshot.getPath();
+			} else if (manualScreenshot.exists()) {
+				backgroundBitmapPath = manualScreenshot.getPath();
```

Comment:
this turns around priority:

previously: manual screenshot exists -> use manual screenshot for backgroundBitmapPath *ELSE* use automatic screenshot.

now: automatic screenshot exists -> use automatc screenshot for backgroundBitmapPath *ELSE* use manual screenshot. so whenever the user takes a manual screenshot it's going to be "overwritten" by the automatic one.

Reply:
exactly the manual screenshot is always going to be overwritten, unless the automatic screenshot is not there or deleted somehow. according to the tickets description.

Reply:
oh, yes. I'm sorry, I completely misunderstood that. I did not realise that this is only used for the visual placement here.